File: redis/golang/models/responses/create_response.go
package modelresponses

import modelentities "note-golang-redis/models/entities"

type CreateResponse struct {
	Id   string `json:"id"`
	Test string `json:"test"`
}

func SetCreateResponse(test1 modelentities.Test1) CreateResponse {
	return CreateResponse{
		Id:   test1.Id,
		Test: test1.Test,
	}
}

File: redis/rust/src/models/responses/create_response.rs
use serde::Serialize;
use crate::models::entities::test1::Test1;

#[derive(Debug, Serialize)]
pub struct CreateResponse {
    pub id: String,
    pub test: String,
}

impl CreateResponse {
    pub fn set_create_response(test1: Test1) -> Self {
        Self { id: test1.id, test: test1.test }
    }
}